lists.html

```diff
@@ -0,0 +1,60 @@
+<!Doctype html>
+<html>
+<head><title>Lists</title></head>
+<body>
+    <!-- type define which type of list you want -->
```

Comment:
please keep the list name as days and months

Reply:
they did not mentioned in that word document